### PR TITLE
Connector: Move models in file per connector

### PR DESCRIPTION
**Pull request by PopDaph**
No changes on the models, just putting them in dedicated files per connector: 
- /connectors/lib/models/index.ts.  -> for Connector & sequelize_conn & ModelId
- /connectors/lib/models/github.ts
- /connectors/lib/models/slack.ts
- /connectors/lib/models/notion.ts
- /connectors/lib/models/google_drive.ts
- /connectors/lib/models/intercom.ts

Note: we could decide to have them all imported and exported from `@connectors/lib/models` as we do in Front. I prefer specifying directly the file hence this proposal but happy to change if we think it would be better 🙇🏻‍♀️

### CODE REVIEW

File: connectors/src/connectors/google_drive/index.ts [hunk 1]
```diff
@@ -7,16 +7,14 @@ import {
...
 import {
-  Connector,
   GoogleDriveConfig,
   GoogleDriveFiles,
   GoogleDriveFolders,
   GoogleDriveSyncToken,
   GoogleDriveWebhook,
-  ModelId,
-  sequelize_conn,
-} from "@connectors/lib/models.js";
```
Comment (PopDaph) [inline connectors/src/connectors/google_drive/index.ts:19]: Do you know why some imports have a .js while the file was a ts? 🙏🏻

Comment (fontanierh) [reply connectors/src/connectors/google_drive/index.ts:19]: That is pretty weird because IIRC we never wrote JS in `connectors`

**Review by fontanierh — DISMISSED** [review]
THANK YOU 🙏

**Review by spolu — COMMENTED** [review]
\o/

**Pull request merged**